streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -813,27 +821,6 @@ void shutdown(final boolean clean) {
                 tasksToCloseDirty.add(task);
             }
         }
-
-        for (final Task task : tasksToCommit) {
```

Comment:
We can actually simplify the standby task shutdown a LOT

Reply:
Makes sense.